docs/DeveloperGuide.md

```diff
@@ -154,6 +154,18 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### tasksOn command
+
+#### Implementation
+
+The tasksOn command parses the user input and generates a `DateTime` object, however the time fields are default values since we only care about the particular Date, the it filters each patient by whether they have a 
+task on the given Date. Each patient themselves filter their TaskList to arrive at the conclusion on whether they have a task on the given Date. Then the Model is set to show the filtered Patients, each with their own TaskList to also 
+only the Tasks on the given Date.
+
+The Sequence diagram below shows the execution of a tasksOn command
```

Comment:
I think you should use `taskOn` instead of taskOn to reflect that it is a command more clearly.

Reply:
Good point!